backend/config.go

```diff
@@ -12,8 +12,13 @@ import (
 )
 
 const (
-	AppURL               = "GF_APP_URL"
-	ConcurrentQueryCount = "GF_CONCURRENT_QUERY_COUNT"
+	AppURL                           = "GF_APP_URL"
+	ConcurrentQueryCount             = "GF_CONCURRENT_QUERY_COUNT"
+	LogUserFacingDefaultError        = "GF_LOG_USER_FACING_DEFAULT_ERROR"
+	SqlRowLimit                      = "GF_SQL_ROW_LIMIT"
```

Comment:
Yeah SQL* I think is best here and across the board (even though it can look a bit weird)

Reply:
👍